### PR TITLE
99 bottles homework

### PR DESCRIPTION
**Pull request by lukecongdon**
Hi Nicole, 

99 bottles is done. I took an iterative approach to it but got it in time. I tried to re-push the slackbot homework too but still see outstanding changes. Need to work on that with you 1:1 I think.

Thanks, Luke

### CODE REVIEW

**Comment by lukecongdon** [comment]
Added random address generator homework. I think I may have done this the hard way. Would be happy to discuss.

File: luke-lukecongdon/hw_1/99_bottles/99bottles.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+*/
+
+// TEST SIX
+
+for(var i = 99; i > 0; i--) {
+    if (i > 1){
+        console.log(i + " bottles of beer on the wall, " + i + " bottles of beer. \n" + "Take one down and pass it around, " + ( i - 1) + " bottles of beer on the wall. \n");
+    } else if (i = 1){
+        console.log(i + " bottle of beer on the wall, " + i + " bottle of beer. \n" + "Take one down and pass it around, no more bottles of beer on the wall. \n \nNo more bottles of beer on the wall, no more bottles of beer. \n" + "Go to the store and buy some more, 99 bottles of beer on the wall. \n");
+    }
```
Comment (Borgaard) [inline luke-lukecongdon/hw_1/99_bottles/99bottles.js:73]: I remember we talked about this elimination of the `else` statement, but I don't remember the reason for just including the last line if `i` didn't meet the criteria of the `if` and `else if`.

Either way, still this solution covered your edge case. 👍

File: luke-lukecongdon/hw_1/madlib-with-loops/js/madlib-console.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+var random1 = Math.floor((Math.random() * startupX.length));
+var random2 = Math.floor((Math.random() * startupY.length));
+
+console.log('A startup that is ' + startupX[random1] + ', but for ' + startupY[random2]);
```
Comment (Borgaard) [inline luke-lukecongdon/hw_1/madlib-with-loops/js/madlib-console.js:7]: You just got this working today during office hours - I'd love to see the solution when you're ready to push it!

File: luke-lukecongdon/hw_1/randomAddress/randomAddress.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+var number = stNum[Math.floor(Math.random()*stNum.length)];
+//    console.log(number);
+// array name function and length must match 
+var name = stName[Math.floor(Math.random()*stName.length)];
+//    console.log(name);
+var city = cityName[Math.floor(Math.random()*cityName.length)];
+//    console.log(city);
+var state = stateName[Math.floor(Math.random()*stateName.length)];
+//    console.log(state);
+var zip = zipCode[Math.floor(Math.random()*zipCode.length)];
```
Comment (Borgaard) [inline luke-lukecongdon/hw_1/randomAddress/randomAddress.js:81]: Explicit functionality in your Math methods AND intuitive variable names? Very nice! :octocat:

File: luke-lukecongdon/hw_7/callback_exercise/main.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        var hours = date.getHours();
+        // Minutes part from the timestamp
+        var minutes = "0" + date.getMinutes();
+        // Seconds part from the timestamp
+        var seconds = "0" + date.getSeconds();
+
+        // Will display time in 10:30:23 format
+        var formattedTime = hours + ':' + minutes.substr(-2) + ':' + seconds.substr(-2);
+        console.log("Time: " + formattedTime);
+      },
```
Comment (Borgaard) [inline luke-lukecongdon/hw_7/callback_exercise/main.js:81]: Are you wanting to work more with time/date data? Once you start feeling comfortable with libraries and APIs, Moment.js is an excellent and well documented time library with examples and starter code.

http://momentjs.com/

**Pull request merged**